src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -155,6 +188,58 @@ private IEnumerable<DependencyDetail> FilterToolsetDependencies(IEnumerable<Depe
             return dependencies;
         }
 
+        private void LogBasicNodeDetails(DependencyGraphNode node, string indent)
+        {
+            Console.WriteLine($"{indent}- Repo:     {node.RepoUri}");
+            Console.WriteLine($"{indent}  Commit:   {node.Commit}");
+            GitDiff diffFrom = node.DiffFrom;
+            // Log the node diff if it was desired.
+            if (diffFrom != null)
```

Comment:
Is this... parsing the output from git? That seems super fragile.  And I don't know if this is something that might run on a users computer, but it will fail if it's not English.

Reply:
No, this is direct from the github/azdo apis internally.